.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - run: cmake --version
       - uses: ilammy/msvc-dev-cmd@v1
       - run: curl -L --output WebKitRequirementsWin64.zip https://github.com/WebKitForWindows/WebKitRequirements/releases/download/v2024.02.05/WebKitRequirementsWin64.zip
-      - run: mkdir build
-      - run: Expand-Archive -Force WebKitRequirementsWin64.zip build/WebKitLibraries
-      - run: ls build/WebKitLibraries
+      - run: mkdir build2
```

Comment:
Can we use a better name than `build2`? Something that better explains how this is different from just `build`?

Reply:
This is addressed by https://github.com/crossnx/includejs/pull/3/commits/82866d3f37373cf64d4a26b9075516ccb194487f

Will rebase and use this PR to update the webkit commit and vendor pull.

Reply:
Keep in mind that we shouldn't merge this at all. These GitHub Actions changes are just for experimenting with WebKit builds. We should then integrate everything purely into CMake